scripts/dream.bash

```diff
@@ -13,11 +13,11 @@ else
     DAQLITE_CONFIG="../configs"
 fi
 
-$DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_tof.json $KAFKA_CONFIG &
-$DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream.json $KAFKA_CONFIG &
-$DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_bwe.json $KAFKA_CONFIG &
-$DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_fwe.json $KAFKA_CONFIG &
-$DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_mantle.json $KAFKA_CONFIG &
-$DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_hr.json $KAFKA_CONFIG &
-$DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_sans.json $KAFKA_CONFIG &
+# $DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_tof.json $KAFKA_CONFIG &
+# $DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream.json $KAFKA_CONFIG &
+# $DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_bwe.json $KAFKA_CONFIG &
+# $DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_fwe.json $KAFKA_CONFIG &
+# $DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_mantle.json $KAFKA_CONFIG &
+# $DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_hr.json $KAFKA_CONFIG &
+# $DAQLITE_HOME/bin/daqlite $BROKER -f $DAQLITE_CONFIG/dream/dream_sans.json $KAFKA_CONFIG &
```

Comment:
These should not be commented away as the default.

Reply:
Thanks for notice. Forgotten after debug.